webpack/assets/javascripts/react_app/components/hosts/PowerStatus.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {Label} from 'react-bootstrap';
+import helpers from '../../common/helpers';
+import HostsStore from '../../stores/HostsStore';
+import HostsActions from '../../actions/HostsActions';
+
+const PowerStates = {
+    on: {
+        text: ('On'),
+        bsStyle: 'default'
+    },
+    off: {
+        text: ('Off'),
+        bsStyle: 'success'
+    },
+    na: {
+        text: ('N/A'),
+        bsStyle: 'info'
+    }
+};
+
+class PowerStatus extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = PowerStates.na; // Sets initiale state to unknown - revisit
+    helpers.bindMethods(this, [
+      'onChange',
+      'onError']
+    );
+  }
+
+  componentDidMount() {
+    HostsStore.addChangeListener(this.onChange);
+    HostsStore.addErrorListener(this.onError);
+    HostsActions.getHostPowerState(this.props.id);
+  }
+
+  componentWillUnmount() {
+    HostsStore.removeChangeListener(this.onChange);
+    HostsStore.removeErrorListener(this.onError);
+  }
+
+  onChange(event) {
+    const id = parseInt(event.id, 10);
+
+    if (id === this.props.id) {
+      const host = HostsStore.getHostsData(id);
+
+      this.updateState(host);
+    }
+  }
+
+  onError(info) {
+    console.log(info);
```

Comment:
I think using console.log makes sense here.

Adding:
```js
// eslint-disable-next-line no-console
```
before the console.log statement will take care of the hound warning